packages/outputarea/src/model.ts

```diff
@@ -153,19 +151,6 @@ export class OutputAreaModel implements IOutputAreaModel {
       });
     }
     this.list.changed.connect(this._onListChanged, this);
-
-    // If we are given a IModelDB, keep an up-to-date
-    // serialized copy of the OutputAreaModel in it.
-    if (options.modelDB) {
```

Comment:
Should we also delete the modelDB option in the options interface?

Reply:
Sure, I guess that's a breaking change, but one that I am pretty sure was used nowhere but in the context of Google RTC.

Reply:
Can you note it above in the breaking changes section?

Reply:
oh, you already did. Thanks!